packages/math-input/src/components/tabbar/__tests__/tabbar.test.tsx

```diff
@@ -1,65 +1,54 @@
-import {mount} from "enzyme";
+import {render, screen} from "@testing-library/react";
```

Comment:
I rewrote all of these tests:
- Most were testing Tabbar state, but Tabbar is stateless now
- I felt bad commenting/removing tests
- Switched to RTL from enzyme